setup.py

```diff
@@ -27,10 +27,6 @@
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Microsoft :: Windows ',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
```

Comment:
The line
```
        'Programming Language :: Python :: 3',
```
should stay because it indicates that it's Python 3 code, at least that's my understanding.